examples/authn/invite_actions.go

```diff
@@ -0,0 +1,102 @@
+// vault sign is an example of how to use the sign/verify methods
```

Comment:
I think we can remove this comment here?

Reply:
Thanks @r3dcrosse! Updated!